<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-RE35JDP4" failures="1" tests="10" name="Test" time="82.955" errors="0" timestamp="12 Mar 2024 15:10:58 GMT">
  <testcase classname="com.sqa.qa.testcases.TransactionsPageTest" name="Verify_Deposit_With_ValideAmmount" time="1.843"/>
  <testcase classname="com.sqa.qa.testcases.CustomerLoginTest" name="customerlogin" time="1.623"/>
  <testcase classname="com.sqa.qa.testcases.CustomersPageTest" name="DeleteCustomerEntry" time="0.881"/>
  <testcase classname="com.sqa.qa.testcases.AddCustomerTest" name="AddCustomer_NotExist" time="1.427"/>
  <testcase classname="com.sqa.qa.testcases.OpenAccountPageTest" name="OpenAccount" time="2.031"/>
  <testcase classname="com.sqa.qa.testcases.AccountPageTest" name="Verify_Deposit_With_ValideAmmount" time="2.106"/>
  <testcase classname="com.sqa.qa.testcases.AddCustomerTest" name="AddCustomer_Exist" time="1.978"/>
  <testcase classname="com.sqa.qa.testcases.AccountPageTest" name="Verify_Deposit_With_InValideAmmount" time="1.133">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.sqa.qa.pages.AccountPage.InValidDeposit(AccountPage.java:176)
	at com.sqa.qa.testcases.AccountPageTest.Verify_Deposit_With_InValideAmmount(AccountPageTest.java:44)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- Verify_Deposit_With_InValideAmmount -->
  <testcase classname="com.sqa.qa.testcases.AccountPageTest" name="Verify_Withdraw_With_ValideAmmount" time="11.938"/>
  <testcase classname="com.sqa.qa.testcases.AccountPageTest" name="Verify_Withdraw_With_InSufficient" time="11.918"/>
</testsuite> <!-- Test -->
